test/scripts/e2e_startup.sh

```
@@ -27,27 +27,46 @@ if [[ -n "${API_ENDPOINT:-}" ]]; then
+            # Check if AAP token is provided for Quadlet AAP authentication
+            if [[ -n "${E2E_AAP_TOKEN:-}" ]]; then
+                echo "🔄 [Startup] Using AAP token authentication for Quadlet environment..."
+                # Login with AAP bearer token
+                set +e
+                LOGIN_OUTPUT=$("${FLIGHTCTL_BIN}" login "${API_ENDPOINT}" --insecure-skip-tls-verify --token "${E2E_AAP_TOKEN}" 2>&1)
+                LOGIN_EXIT=$?
+                set -e
+                if [[ "${LOGIN_EXIT}" -ne 0 ]]; then
+                    echo "⚠️  [Startup] Warning: Could not login to API at ${API_ENDPOINT} with AAP token"
+                    echo "    Login error output:"
+                    echo "${LOGIN_OUTPUT}" | sed 's/^/    /'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)